packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -97,11 +99,9 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
             getChildNodes(key).map((node, index) => (
               <React.Fragment key={index}>
                 <SceneHierarchyTreeItem key={node.objectRef} enableDragAndDrop={enableDragAndDrop} {...node} />
-                {showSubModel && node?.componentTypes?.includes(KnownComponentType.SubModelRef) && (
-                  <SubModelTree parentRef={key} expanded={false} object3D={model!} selectable />
-                )}
               </React.Fragment>
             ))}
+          {showSubModel && <SubModelTree parentRef={key} expanded={false} object3D={model!} selectable />}
```

Comment:
Just a heads up that my incoming change will move this back into the `React.Fragment` with fixes for hierarchy placement. Non-blocking, but please let me know if you have any concerns with this regarding your fix.

Reply:
we shouldn't put it into the Fragment because this tree is for the root ModelRef node, not for the child SubModelRef nodes. the current code breaks submodel feature because when I create a new scene and add a model to it, I won't be able to see the tree and cannot add sub model ref component to the scene, so this tree never show up